docker-compose.yml

```diff
@@ -15,3 +15,5 @@ services:
       - "17474:7474"
     volumes:
       - ./db/dbms:/data/dbms
+    environment:
+      - NEO4J_AUTH=none
```

Comment:
its easier than configuring a password manually, and not a problem when running locally